packages/core/src/styled/Chips/styles/index.ts

```diff
@@ -0,0 +1,180 @@
+import {
+    KeysFromUseStyles,
+    makeStyles, transitions, typography,
+} from '@core/styles';
+import { cssVar } from '@core/utils/cssVars';
+import { paramsToCss } from '@core/utils/paramsToCss';
+import { QX_SIZE } from '@core';
+import { CHIPS_VARIANT } from '@core/src/styled/Chips/constants';
+import { ChipsCSSVarKeys } from './vars';
+import { ChipsStyleParams } from './types';
+
+export const useStyles = makeStyles((
+    { palette, elevations },
+    {
+        elevation,
+        active,
+        size,
+        leftIcon,
+        rightIcon,
+        disabled,
+        disableFocus,
+        variant,
+    }: ChipsStyleParams,
+    {
+        cssLeftIconColor,
+    }: Record<ChipsCSSVarKeys, string>,
+) => {
+    const notAllowed = {
+        cursor: 'not-allowed',
+    };
+
+    const press = {
+        '&:active': !disabled && {
+            backgroundColor: palette.colors.brand.alpha[16],
+        },
+    };
+
+    const elevationOn = {
+        boxShadow: elevations.small,
+        '&:hover': !disabled && {
+            backgroundColor: palette.colors.brand.alpha[8],
+            boxShadow: elevations.medium,
+        },
+        ...press,
+        '&:focus-visible': !disabled && !disableFocus && {
+            borderColor: 'transparent',
+            boxShadow: `0px 0px 0px 3px ${palette.border.focus.dark}, ${elevations.medium}`,
```

Comment:
здесь и ниже
```suggestion
            boxShadow: `inset 0px 0px 0px 3px ${palette.border.focus.dark}, ${elevations.medium}`,
```

А вообще предлагаю обводку фокуса теперь создавать иначе. В `Switcher`, `RadioButton` и `Button` (если обновишь `main`-ветку) есть примеры использования `baseFocusStyles` и `stylesWithFocus`.